### PR TITLE
New filters: an participating organisation that is specifically funding/implementing etc.

### PR DESCRIPTION
**Pull request by odscjames**
https://github.com/codeforIATI/iati-datastore/issues/34

### CODE REVIEW

**Review by markbrough — COMMENTED** [review]
I think this is ready to go, no? @odscjames

File: iati_datastore/iatilib/test/test_filter.py
Comment (markbrough) [inline iati_datastore/iatilib/test/test_filter.py:837]: If we are testing whether the filters only apply to participating orgs with the relevant `role`, perhaps this should be
```python
organisation__ref=u"AAA",
role="2"
```

**Review by markbrough — COMMENTED** [review]
Thanks for adding these tests! One question but will leave it up to you to decide. Then I think good to go when you're ready!

File: iati_datastore/iatilib/test/test_filter.py
Comment (odscjames) [reply iati_datastore/iatilib/test/test_filter.py:843]: we should test both, done

**Review by markbrough — APPROVED** [review]
Looks great, thank you!

**Pull request merged**
